docs/lib/content/configuring-npm/package-lock-json.md

```diff
@@ -114,10 +114,10 @@ the npm registry.  Lockfiles generated by npm v7 will contain
 * `1`: The lockfile version used by npm v5 and v6.
 * `2`: The lockfile version used by npm v7, which is backwards compatible
```

Comment:
This is the version "used" by npm v7 and v8.  "used" in this context means the one configured by default.

Reply:
Just to be sure, would you like me to include in this commit a change of "by npm v7" to "by npm v7 and v8" here?

Reply:
yes

Reply:
Done.